database-mongodb/index.js

```diff
@@ -75,11 +75,11 @@ const getDescriptionObject = (itemId) => {
 };
 
 const postDescriptionObject = (descObj) => {
-  return Description.create(descObj);
+  return Description.findOneAndUpdate({ itemId: descObj.itemId }, descObj, {upsert: true});
```

Comment:
Why did this get changed from create to findoneandupdate? Isn't one more for a post request and one is for a put request?

Reply:
this was quite the rabbit hole, but the problem I was having was that create pushes the posted document into an array of documents.  I really never liked that.  there were some other unexpected behaviors too, but the long and short of it is that i'd rather just upsert for this method.

Reply:
actually you just gave me an idea and i think it won't matter either way.  going to revert back to create, but just return out of the route if there's a conflict.  that way it won't run if the post exists.  thanks Will!